lib/Cron/CleanupSessions.php

```diff
@@ -45,9 +46,10 @@ public function __construct(ITimeFactory  $time,
 								SessionMapper $sessionMapper) {
 		parent::__construct($time);
 		$this->sessionMapper = $sessionMapper;
+		$this->config = $config;
 		// daily
 		$this->setInterval(24 * 60 * 60);
-		$this->config = $config;
+		$this->setTimeSensitivity(IJob::TIME_INSENSITIVE);
```

Comment:
May need a method_exists check if we want to keep the app compatible below Nextcloud 24.

Reply:
The method is available in server's stable23 and stable22 (even if @since says different :grin:). So it would just be for NC 21. We can also drop support for NC 21. What do you think?

Reply:
Oh right, we back ported it, but may still throw then on older patch releases

Reply:
Yep